lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -162,5 +163,20 @@ sub parent_group_overview {
     $self->group_overview('JobGroupParents', 'main/parent_group_overview');
 }
 
+sub changelog {
+    my ($self) = @_;
+
+    my $changelog;
+    if(open(my $changelog_file, '<', '/usr/share/openqa/ChangeLog')) {
```

Comment:
once [this](https://build.opensuse.org/request/show/451342) gets accepted into OBS, the Changelog will be in public/Changelog